.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -1,6 +1,9 @@
 ---
-name: Bug report
-about: Report errors and problems
+name: "\U0001F41E Bug Report"
+about: Did you find a bug?
+title: ''
+labels: ''
```

Comment:
any reason to not add the bug label here? I assume that will auto-add the label to the issue?

Reply:
Ah, good catch, thanks.